skbio/tree/_tree.py

```diff
@@ -2705,6 +2705,47 @@ def compare_tip_distances(self, other, sample=None, dist_f=distance_from_r,
         return dist_f(self_matrix, other_matrix)
 
     @experimental(as_of="0.4.0")
+    def bifurcate(self, insert_length=None, remove_singles=True):
+        r""" Restructures tree into a bifurcating tree
+
+        All nodes that have more than 2 children will
+        have additional intermediate nodes injected to ensure that
+        every node has only 2 children
+        Also, any nodes that have a single child will be collapsed
+
+        Parameters
+        ----------
+        remove_singles : bool, optional
```

Comment:
So that was one thing that I was a little hung up on when putting together this method.  Because I wanted to enable this method to allow for both bifurcating trees (internal nodes have at most 2 children) and strictly bifurcating trees (internal nodes have strictly 2 children)

I'm fine with removing the `remove_singles` parameters.  However, would it become more unintuitive to create strictly bifurcating trees for the average user?

To create a strictly bifurcating tree, it would basically change the following command from
```
tree.bifurcate(remove_singles=True)
``` 
To
```
tree.bifurcate()
tree.prune()
```

In hindsight, I think it is ok to remove this parameter